Sources/RInAppMessaging/Views/Presenters/FullViewPresenter.swift

```diff
@@ -1,6 +1,12 @@
 import UIKit
 import UserNotifications
 
+#if canImport(RSDKUtils)
+import class RSDKUtils.AnalyticsBroadcaster
+#else // SPM Version
+import class RSDKUtilsMain.AnalyticsBroadcaster
+#endif
```

Comment:
nit: it's only my opinion but it's maybe clearer to use `#if SWIFT_PACKAGE/#else/#endif`.

Reply:
Oh yeah that's a good suggestion. I agree. I will create another PR with that change in the whole module